demisto_sdk/commands/validate/file_validator.py

```diff
@@ -392,6 +374,7 @@ def validate_modified_files(self, modified_files):  # noqa: C901
 
             elif 'CHANGELOG' in file_path:
                 self.is_valid_release_notes(file_path)
+                continue
```

Comment:
why is that `continue` needed? 
if im not missing something, it would continue to the next iteration anyway

Reply:
Forgot to remove. will do